echopype/tests/visualize/test_plot.py

```diff
@@ -173,6 +173,10 @@ def test_plot_mvbs(
     azfp_xml_path,
     range_kwargs,
 ):
+
+    # pytest.xfail("Test momentarily (2022-9-9, EM) set to xfail due to 'Killed' failure")
```

Comment:
@emiliom can you explain this comment and how it relates to this PR?

Reply:
This was an oversight and should be removed. I was dealing with some test failures, but forgot to remove this line. I'll remove it here.